### PR TITLE
Provide example for OpenAPI and Swagger UI usage

### PR DESCRIPTION
**Pull request by Voycawojka**
## Description
Example usages of more advanced OpenAPI features and Swagger UI in Knot.x

## Motivation and Context
There's no such example

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the [code style](/CONTRIBUTING.md#coding-conventions) of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

---
I hereby agree to the terms of the Knot.x Contributor License Agreement.

### CODE REVIEW

File: api-gateway/api-console/knotx/conf/logback.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <layout class="ch.qos.logback.classic.PatternLayout">
+      <Pattern>
+        %d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n
+      </Pattern>
+    </layout>
+  </appender>
+
+  <logger name="io.knotx" level="TRACE"/>
+  <logger name="io.vertx" level="TRACE"/>
+  <logger name="com.codahale.metrics" level="TRACE"/>
```
Comment (oskarjerzyk) [inline api-gateway/api-console/knotx/conf/logback.xml]: Do we really need trace logging level here?

File: api-gateway/api-console/modules/swagger/src/main/java/io/knotx/example/swagger/handler/AdminInfoHandlerFactory.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    return routingContext -> {
+      RequestContext requestContext = routingContext.get(RequestContext.KEY);
+      ClientRequest clientRequest = requestContext.getRequestEvent().getClientRequest();
+
+      String type = clientRequest.getParams().get("datatype");
+      respondByType(routingContext.response(), type);
+    };
+  }
+
+  private void respondByType(HttpServerResponse response, String type) {
```
Comment (oskarjerzyk) [inline api-gateway/api-console/modules/swagger/src/main/java/io/knotx/example/swagger/handler/AdminInfoHandlerFactory.java]: Is it possible that response will be in yaml?

Comment (Voycawojka) [reply api-gateway/api-console/modules/swagger/src/main/java/io/knotx/example/swagger/handler/AdminInfoHandlerFactory.java]: I didn't include yaml here for two reasons
- two types are enought to demonstrate the concept
- yaml is more suited for the configuration files etc. I don't think it's commonly used for client-server configuration

File: api-gateway/api-console/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+[![][license img]][license]
+[![][gitter img]][gitter]
+
+# Developer API Console
+
+This project provides an example for a more advanced uses of the OpenAPI standard.
```
[the remark this answers is not in the document]
Comment (Voycawojka) [reply api-gateway/api-console/README.md]: I think `usages` is also correct but sure

File: api-gateway/api-console/functional/src/test/java/com/project/test/functional/ExampleApiITCase.java [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import static io.restassured.RestAssured.given;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ExampleApiITCase {
+
+  @Test
+  @DisplayName("Expect 200 status code when endpoint registered.")
+  void callUndefinedRoute() {
```
[the remark this answers is not in the document]
Comment (tomaszmichalak) [reply api-gateway/api-console/functional/src/test/java/com/project/test/functional/ExampleApiITCase.java]: This will be delivered in a separate PR.

**Pull request merged**